src/equations/ideal_glm_mhd_multicomponent_1d.jl

```diff
@@ -0,0 +1,469 @@
+
+@doc raw"""
+    IdealGlmMhdEquations1D
```

Comment:
```suggestion
    IdealGlmMhdMulticomponentEquations1D
```
Haven't I already commented on this in a previous review?

Reply:
Yes, but just for 2D 🐙. Done in 0539fb8